ksqldb-engine/src/test/java/io/confluent/ksql/integration/EndToEndIntegrationTest.java

```diff
@@ -255,6 +259,73 @@ public void shouldSupportDroppingAndRecreatingJoinQuery() throws Exception {
     assertThat(columns.get(2).toString(), either(is("FEMALE")).or(is("MALE")));
   }
 
+  @Test
+  public void shouldSupportPauseAndResume() throws Exception {
```

Comment:
What does this test offer beyond the new integration tests in PauseResumeIntegrationTest.java? If nothing, we can remove this one. It'd be nice to keep all the integration testing for the feature in one place.